### ISSUE TITLE
Add an example to visualize framing in news

### ISSUE THREAD

**Issue by monajalal** [davidatbu/openFraming#328]
<img width="597" alt="Screen Shot 2020-09-17 at 12 21 56 AM" src="https://user-images.githubusercontent.com/1892917/93420162-6dcce880-f87c-11ea-83e6-d3d1f81fbea4.png">

Here: http://www.openframing.org/

**Comment by monajalal** [comment]
@edward-edberg could you work with Yimeng on this? We would like this (a nicer way of showing it though not screenshot) to be first thing when you browse to openframing.org so you get an idea what framing (e.g. in news) means in general.

**Comment by edward-edberg** [comment]
@monajalal do you mean just add an image?

**Comment by monajalal** [comment]
@edward-edberg yes please add the image that @yimengsun created and please ask @davidatbu to push this to deploy

**Comment by edward-edberg** [comment]
https://github.com/davidatbu/openFraming/pull/331
done

Sorry for late reply, my mom passed away 2 days ago. I still cannot concentrate properly.

**Comment by monajalal** [comment]
@edward-edberg  really sorry for your loss. Please accept my condolences.

@davidatbu could you please submit these pushes to the actual website and also provide us guidelines how to do this on our own in the future perhaps?

**Comment by davidatbu** [comment]
Thank you for the PR Edward. I appologize for the silence on my end. I've deployed the latest code.

@monajalal , to deploy,log in to the instance, and then
```bash
cd openFraming/
git status # Just to make sure  that we're on the deployment branch
git pull # Pull the latest changes from the deployment branch
sudo docker-compose build # Build the Docker image according to the latest repo
sudo docker-compose up - d #  (Re)start the docker containers to use the latest image.
```

Note that the `.env` file on the instance has the SendGrid API keys (as it states to do so on the README). These cannot be committed to the repo because that would make them public.
